atlas_densities/densities/mtype_densities_from_map/create.py

```diff
@@ -10,141 +10,23 @@
-            "layer_5_sst") and whose columns are labelled by mtypes (morphological types, e.g.,
-            "chc", "lac").
-        layer_names: set of layer names specified in the metadata json file.
-        molecular_types: set of molecular types a.k.a gene marker types,
-            e.g., "pv", "sst", "vip", "gad67", "htr3a".
-    """
-    if "layer_1_gad67" not in probability_map.index:
-        raise AtlasDensitiesError(
-            "Missing probabilities for the molecular type gad67 in layer 1: "
-            "'layer_1_gad67' was not found in data frame index."
-        )
-
-    check_probability_map_sanity(probability_map)
-
-    df_layer_names, endings = zip(*[row.rsplit("_", 1) for row in probability_map.index])
-    diff = molecular_types - set(endings)
-    if diff:
-        raise AtlasDensitiesError(
-            f"The following molecular types are missing in the probablity map: {diff}"
-        )
-    diff = set(df_layer_names) - layer_names
-    if diff:
-        raise AtlasDensitiesError(
-            f"Unexpected layer names in probability map: {diff}. "
-            f"Expected (from metadata json file): {layer_names}"
-        )
-
-
-def _get_coefficients(
-    mtype: str,
-    probability_map: "pd.DataFrame",
-    layer_names: List[str],
-    molecular_types: List[str],
-) -> Dict[str, Dict]:
-    """
-    Get from `probability_map` the coefficients used to express the `mtype` density as
-    a linear combination of the input molecular type densities.
-
-    This function assumes a good agreement exists between the layer names used in `metadata`
-    and those used in row labels of `probability_map`.
-
-    Args:
-        mtype: the morphological type for the coefficients of which are queried.
-        probability_map: data frame whose rows are labelled by molecular types and layers
-            (e.g., "layer_23_pv", "layer_5_sst") and whose columns are labeled by mtypes
-            (morphological types, e.g., "chc", "lac").
-        layer_names: layer names according to the metadata json file and `probability_map`.
-            Assumption: they coincide and the first layer name is "layer_1".
-        molecular_types: molecular types, e.g., "pv", "sst", "vip", "gad67", with an extra
-            value "rest" which denote the type of the neurons reacting to gad67 but not to
-            any of the markers pv, sst or vip.
-
-    Returns:
-        dict whose keys are layer names and whose values are dicts mapping molecular types
-        to float coefficients in the range [0.0, 1.0].
-    """
-    coefficients = {}
-    for layer_name in layer_names[1:]:
-        coefficients[layer_name] = {
-            molecular_type: probability_map.at[f"{layer_name}_{molecular_type}", mtype]
-            for molecular_type in molecular_types
-            if molecular_type != "gad67"
-            and not (layer_name == "layer_6" and molecular_type == "vip")
-        }
-
-    return coefficients
+logger = logging.getLogger(__name__)
```

Comment:
we have really standardized on `L` being the name of the logger, so please don't change that.

Reply:
that is rather unfortunate, capital letters are used for classes and on top of that it is a single letter, I remain very unhappy about this standardization